plugin/storage/kafka/options_test.go

```diff
@@ -164,3 +167,43 @@ func TestRequiredAcksFailures(t *testing.T) {
 	_, err := getRequiredAcks("test")
 	assert.Error(t, err)
 }
+
+func TestTLSFlags(t *testing.T) {
```

Comment:
These tests are duplicated. Don't we have Kafka flags parsed by a packaged shared between ingester and storage?

Reply:
Both tests are in different packages, there is no shared test class.